net-misc/unison/unison-2.51.5.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop xdg-utils
+
+DESCRIPTION="Two-way cross-platform file synchronizer"
+HOMEPAGE="https://www.seas.upenn.edu/~bcpierce/unison/"
+SRC_URI="https://github.com/bcpierce00/unison/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="$(ver_cut 1-2)"
+KEYWORDS="~amd64 ~arm ~ppc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~sparc-solaris"
+IUSE="debug doc gtk +ocamlopt threads"
+
+# ocaml version so we are sure it has ocamlopt use flag
+BDEPEND="dev-lang/ocaml:=[ocamlopt?]
+	doc? ( app-text/dvipsk
+			app-text/ghostscript-gpl
```

Comment:
The indentation here is changed, compared with the previous version's ebuild. Please rectify that.

Reply:
Thanks for the feedback. I fixed that.